source/jsDecode.brs

```diff
@@ -122,6 +122,11 @@ Function getOtherFuncs(primary_func as Object, jsText as String) as Object
     '""" Return all secondary functions used in primary_func. """
     'print("scanning javascript for secondary functions.")
     body = primary_func.body
+    
+    if ( body = invalid ) then
```

Comment:
I've seen that error in the past, were you running the latest 'developer version'? (not the full release)